src/main/java/CommandClasses/Command.java

```diff
@@ -0,0 +1,100 @@
+package CommandClasses;
+
+import ExceptionClasses.EmptyByOrAtWhenException;
+import ExceptionClasses.EmptyTaskDescriptionException;
+import ExceptionClasses.TooManyArgumentsException;
+
+public class Command {
+    private CommandEnum commandType;
+    private String description;
+    private String when;
```

Comment:
This name could be improved further by considering if it is referring to a [noun or verb].